### DISCUSSION TITLE
Can quartz support some certain server in a cluster NOT to run jobs ?

### DISCUSSION THREAD

**Discussion by jianglingyuea** [quartz-scheduler/quartz#1264] [Q&A] [answered]
I have imported Quartz into a cluster with 3 servers. I want a certain server not to run Quartz jobs, but I still want to be able to create jobs on this server and have the other servers run these jobs.
I have tried to throw a JobExecutionException on the certain server, but it seemed that this server can always get the lock and fail on the job again and again while other servers cannot run this job.

**Comment by jhouserizer** [comment, marked as answer]
Yes, this is covered in the docs - simply initialize the scheduler on that node, but do not call .start() on it.

**Reply by jianglingyuea** [reply]
Thanks for your help. But didn't call .start() in my program. Actually, I just create a job via scheduler.scheduleJob(jobDetail, trigger) and the server will automatically handle this job. In the cluster mode, random server will handle this job.  How can I make certain server not handle jobs in such a situation?

**Reply by jianglingyuea** [reply]
<img width="980" alt="image" src="https://github.com/user-attachments/assets/b53578f2-1e9a-4b3c-88f7-966f7872c8f2">

**Reply by jianglingyuea** [reply]
And the scheduler is autowired in the controller 
<img width="536" alt="image" src="https://github.com/user-attachments/assets/ec3e09e7-97bd-4ebc-a8a6-783044666283">

**Reply by defrimhasani** [reply]
@jianglingyuea looks like you are using spring-quartz, most likely with some quartz properties in your yaml or properties. On a given node, you can add the following and the scheduler won't start, if you are running in clustered node, you can check the scheduler_state table to verify that the node is not there.

```yaml
spring:
  quartz:
    auto-startup: false
```
